src/browser/commands/types.ts

```diff
@@ -87,8 +88,9 @@ export type AssertViewCommand = (
     selectors: string | string[],
     opts?: AssertViewOpts,
 ) => Promise<void>;
+
 export type AssertViewElementCommand = (
-    this: WebdriverIO.Element,
+    this: WebdriverIO.Element | ChainablePromiseElement<WebdriverIO.Element>,
```

Comment:
In order to be able write code like - `await browser.$("foo").assertView("plain")`. Without this I should write it like this - `await (await browser.$("foo")).assertView("plain");`